docs/src/main/sphinx/sql/show-session.rst

```diff
@@ -13,9 +13,14 @@ Description
 -----------
 
 List the current :ref:`session properties <session-properties-definition>`.
-The ``LIKE`` clause can be used to restrict the list of session properties.
 
-See also
---------
+:ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to
+filter the results to the desired subset. For example, the following query
+allows you to find session properties that begin with ``query``::
+
+    SHOW SESSION LIKE 'query%'
+
+See related syntax
```

Comment:
Why the title change here? lets undo that ..

Reply:
I'll revert it because it isn't related to the task, but "See also" sounds weird. Plus, it's an incomplete action, imo. I think it should say something like, "See more session commands".